my-app/src/App.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react'
+import logo from './logo.svg'
+import './App.css';
+
+class Board extends Component {
+  constructor() {
+    super();
+    this.size = 9;
```

Comment:
In React, we should be provided data about a component via props, or record data in the component's state.  I would create a constant outside of this component, that would be used to pass in the size as a prop.  Remember that the object that `this` refers to is not a game board - it's a React component, so it doesn't make sense to create an attribute on the component for `size`.